pytest_plugins/fixture_collection.py

```diff
@@ -22,17 +22,18 @@ def pytest_collection_modifyitems(items, config):
         return
 
     filter_fixtures = config.getvalue('uses_fixtures')
+    fixtures_list = filter_fixtures.split(',') if ',' in filter_fixtures else [filter_fixtures]
```

Comment:
Shouldn't we support both spaces and commas for uses_fixtures here? 
```suggestion
    filter_fixtures = config.getvalue('uses_fixtures')
    fixtures = ' '.join(filter_fixtures.split(',')) if ',' in filter_fixtures else filter_fixtures
```

Reply:
Nop only comma, to support the space cli provides the way already which I m removing in the PR as its not working as its suppose to work.

And adding comma separated support.